tortoise/verifying.go

```diff
@@ -76,13 +75,13 @@ func (v *verifying) verify(logger log.Log, lid types.LayerID) bool {
 		return false
 	}
 
-	margin := util.WeightFromUint64(0).
-		Add(v.totalGoodWeight).
+	margin := v.totalGoodWeight.
 		Sub(layer.verifying.goodUncounted)
 	uncounted := v.expectedWeight(v.Config, lid).
 		Sub(margin)
-	if uncounted.Sign() > 0 {
-		margin.Sub(uncounted)
+	// GreaterThan(zero) returns true even if value with negative sign
```

Comment:
curious. is this a bug in fixed64 library?

Reply:
i want to understand that as well)